### PR TITLE
Add recipe for TargQC

### PR DESCRIPTION
**Pull request by vladsaveliev**
* [x] I have read the guidelines above.
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

@bioconda/core please review.

### CODE REVIEW

File: recipes/targqc/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  name: targqc
+  version: {{ version }}
+
+source:
+  fn: targqc-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/e0/98/f95a514128e3ee1bfc60a4a9544df9a0497b691383bfebf991973e35c414/TargQC-1.4.4.tar.gz
+  md5: 75b056abdf2e2cf9860eaa8758ab236a
+
+build:
+  number: 1
```
Comment (bgruening) [inline recipes/targqc/meta.yaml]: for a new package, this can start with 0

**Comment by bgruening** [comment]
@vladsaveliev looks good, I have added one small comment.
Thanks!

**Comment by vladsaveliev** [comment]
Thanks, done!

**Comment by vladsaveliev** [comment]
@bioconda/core, is it good to be merged?

**Comment by daler** [comment]
Looks good!

**Pull request merged**